.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  prodsec: snyk/prodsec-orb@1.0
+  prodsec: snyk/prodsec-orb@1.1
```

Comment:
Question: Is there a reason for upgrading the version here?

Reply:
yes, the pipeline _Scan for hardcoded secrets_ job fails without the orb upgrade on some `git error encountered, see logs` - git connectivity based.